components/sidebar/sidebar.jsx

```diff
@@ -221,7 +225,7 @@ export default class Sidebar extends React.PureComponent {
         this.updateTitle();
 
         // Don't modify favicon for now: https://mattermost.atlassian.net/browse/MM-13643.
```

Comment:
Can you remove this comment since it's incorrect now?

Reply:
Sure